src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentEditAction.java

```diff
@@ -96,9 +103,13 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                            + "by: " + feedbackResponseComment.giverEmail + "<br>"
                            + "comment text: " + feedbackResponseComment.commentText.getValue();
         }
-
-        data.comment = feedbackResponseComment;
-
+        data.comment = updatedComment;
```

Comment:
`//comment fields:- createdAt, lastEditedAt and commentGiverName are required to generate editedCommentDetails.`

will this be fine?